Applications/spire/source/book_view/book_view_highlight_properties_widget.cpp

```diff
@@ -222,10 +228,9 @@ void BookViewHighlightPropertiesWidget::update_color_button_stylesheet(
   button->set_focus_style(s);
 }
 
-void BookViewHighlightPropertiesWidget::update_market_widgets(
-    int selected_item_index) {
+void BookViewHighlightPropertiesWidget::update_market_widgets() {
   auto& selected_item = static_cast<MarketListItem*>(
```

Comment:
Change auto& to auto. This should actually be a compiler error but MSVC is dumb.